examples/metalk8s-solution-example/ui/src/containers/Layout.js

```diff
@@ -87,13 +73,36 @@ const Layout = props => {
     ]
   };
 
+  const rightActions = [
+    {
+      type: 'dropdown',
+      icon: <i className="fas fa-th" />,
+      items: applications
```

Comment:
same here, you should check if there is applications not before adding this rightActions

Reply:
I am wondering in solution-example UI do we need applications item in `Navbar`?